OsmIntegrator/Database/Models/DbTileExportReport.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using OsmIntegrator.Database.Models;
+using OsmIntegrator.Database.Models.JsonFields;
+
+namespace OsmIntegrator.Database.Models
+{
+  [Table("TileExportReport")]
```

Comment:
I have the rule in the project that the DB table name ends with `s`. So if we have a class `DbConnection` the DB table name is called `Connections`.

Reply:
renamed